CONTRIBUTING.md

```diff
@@ -127,42 +172,32 @@ changes on.
 If you're making a change to a current release, run:
 
 ```bash
-git checkout hotfix/<current-release>
-git pull
+git fetch upstream
+git checkout -b <new-branch-name> upstream/hotfix/<current-release>
+make setup
```

Comment:
issue: there is no "setup" makefile target in craft-parts (the project is not yet fully migrated to starbase). Maybe reference the instructions above to install dev deps in the venv?